tests/tel2puml/main/test_main.py

```diff
@@ -221,6 +224,59 @@ def test_handle_exception(
         "ERROR: Use the -d flag for more detailed information." in captured.out
     )
     assert (
-        "An unexpected error occurred. Unexpected test error. Please"
-        " contact smartDCSIT support for assistance." in captured.out
+        "Please contact smartDCSIT support for assistance." in captured.out
     )
+
+
+def test_main(
```

Comment:
can we test for a ValidationError, JSONDecodeError and just plain Exception as well